src/core/utils/expressionutils.h

```diff
@@ -0,0 +1,58 @@
+/***************************************************************************
+ expressionutils.h - ExpressionUtils
+                              -------------------
+ begin                : January 2020
+ copyright            : (C) 2020 by David Signer
+ email                : david (at) opengis.ch
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef EXPRESSIONUTILS_H
+#define EXPRESSIONUTILS_H
+
+#include <qgsexpression.h>
+#include <qgsexpressioncontext.h>
+#include <QObject>
+
+class ExpressionUtils : public QObject
```

Comment:
I think this class should be called "ExpressionEvaluator" or similar, it's more specified than a generic Utils class.

Reply:
but keep it in the utils directory?

Reply:
Not necessarily, no.